job-runner.toml

```diff
@@ -0,0 +1,37 @@
+# Install to ~/.config/cockpit-dev/job-runner.toml
+
+[container]
+command = 'podman'
+args = []
+run-args = ['--device=/dev/kvm']
+default-image = 'quay.io/cockpit/tasks:latest'
+
+[container.secrets]
+# see podman-secret(1)
+# github-token = '--secret=github-token'
```

Comment:
As every human dev needs ~/.config/cockpit-dev/github-token token anyway, let's add an example how to use a normal file-based one? TBH I don't really see the point of creating a podman secret for this. It's just duplication and extra work to set it up.

I have
```
github-token='--volume=/home/martin/.config/cockpit-dev/github-token:/run/secrets/webhook/.config--github-token:U,ro'
```

and this is actually quite awkward: There is little choice about the host path (and moreover, this doesn't accept `~` or `$XDG_CONFIG_DIR`), and our infra already defines where the container expects the secret.

I know this has to be set on prod infra (`/var/lib/cockpit-tasks/secrets/...`), but that can happen in cockpituous `install-service`.

Reply:
Two reasons against this:
 - the human user may well want to use job-runner with a different token than they use for themselves.  For me that's my personal GitHub account vs. `cockpit-copilot` which I use as my testing account, for example.  One of those has admin access to all things cockpit, and the other (as a testing account) very much does it.
 - the usual issues with selinux labels

Reply:
Hmkay (you have a high privileged .config/cockpit-dev/github-token ? That sounds highly dangerous..)

Could you add a full example then?